### ISSUE TITLE
No downlaod for LineageOS/F-Droid Users. Need to delete Brave.

### ISSUE THREAD

**Issue by vsatmydynipnet** [brave/brave-browser#7027]
We are using Brave and Firefox on all Linux and LineageOS (Android) Systems. But we setup without Gapps, so no Google Play is available. We also have no Google Account and do not use their services.

Brave was downloadable via Yalp from Play since a few days and it was possible to work that way without Google Account for the 2 or 3 Apps we use from their store. 

Now Google blocks Yalp, so there is no possibility to upgrade Brave from 1.4.x to latest version.

All others have a direct APK download, Mozilla/Firefox  via their server, Tor via their server or Guardian Project via F-Droid, so working without Goggle and Google Play works perfect. 

Before we delete Brave from all of our Desktop and Mobile Systems there is this question for some download from a trusted server. Maybe possible to add as F-Droid repository would be great.

Thank you.

**Comment by bsclifton** [comment]
cc: @SergeyZhukovsky @samartnik @AlexeyBarabash

**Comment by AlexeyBarabash** [comment]
@vsatmydynipnet 

In the future we will release browser for Android from the same codebase as the desktop browser, and then apk will be possible to find at https://github.com/brave/brave-browser/releases .

AT the current you can find the latest Brave Browser Android apk in `brave/browser-android-tabs` repository, https://github.com/brave/browser-android-tabs/releases . The latest version is https://github.com/brave/browser-android-tabs/releases/tag/v1.5.2 .

**Comment by kjozwiak** [comment]
@vsatmydynipnet just make sure you download a build that's being released. Sometimes builds get published for QA that might have issues. We've had a few instances of users grabbing a build that was still being developed because they thought it was an "official" release.

Usually builds that are uploaded to GPS have release notes associated with them as per https://github.com/brave/browser-android-tabs/releases/tag/v1.6.0. You can also use https://github.com/brave/browser-android-tabs/blob/base-78.0.3904.108/CHANGELOG.md to see the what the latest release is.

**Comment by genodeftest** [comment]
Isn't this a duplicate of #6217?

If you want to have Brave in an unofficial F-Droid repo, please have a look at #9316.

**Comment by srirambv** [comment]
+1 from @paulvi and others via #6217

**Comment by srirambv** [comment]
+1 from @stevenleeS0ht via #9682

**Comment by stevenleeS0ht** [comment]
In F-Droid, the App can be automatically update. It would take more time to do every installation manually.

**Comment by stevenleeS0ht** [comment]
You can push all formal apk release to F-Droid repository automatically via CD script (Jenkins, Travis-CI or some others).

**Comment by stefankeys** [comment]
No chance for an official f-droid repo brave-browser?

**Comment by Flygrounder** [comment]
What's the status of this issue? Is anyone working on this?

**Comment by Krusty27** [comment]
Any update? We would really like to switch to Brave on Android

**Comment by davidhaymond** [comment]
I would also like to see Brave added to F-Droid.

**Comment by bsclifton** [comment]
@Krusty27 @davidhaymond this is labeled as P5 which means no work is planned on our side for this ☹️

**Comment by stefankeys** [comment]
> @Krusty27 @davidhaymond this is labeled as P5 which means no work is planned on our side for this

Disappointing but not surprising at all.

**Comment by Dev-i-l** [comment]
> 
> 
> @Krusty27 @davidhaymond this is labeled as P5 which means no work is planned on our side for this ☹️

this is very much needed feature as f droid seems to be the place where all open source freaks go, brave has been a great project with no user tracking but it being not available on f droid pushes me to use fennec which has tracking built into it. I hope that you people will re-evaluate the neccessity of this feature. Keep up the brave work :)

**Comment by hefty-fading-culprit-epiphany-drew** [comment]
Hope they will work on this someday

**Comment by Cliveptr** [comment]
Relevant F-Droid RFP issue: https://gitlab.com/fdroid/rfp/-/issues/154

Most of the blockers (binaries) seem to be for testing, and are not included in the build. Not sure whether or not that means it'd be eligible for inclusion.

**Comment by Luc45** [comment]
We install Brave to get away from Google, and we have to download it from Play Store :laughing:

Acid jokes aside, it would be really good if Brave made it's way into F-Droid.

**Comment by stefankeys** [comment]
Brendan Eich said something will be done about this: https://twitter.com/BrendanEich/status/1387771839099904007?s=20

**Comment by WilliamShockleyfake** [comment]
> @Krusty27 @davidhaymond this is labeled as P5 which means no work is planned on our side for this ☹️ 

Still?

**Comment by Mhowser** [comment]
Please can we get a higher priority on this?

**Comment by erazemk** [comment]
> Please can we get a higher priority on this?

The issue has been opened more than a year and a half ago and there hasn't been any progress, so they don't seem very interested to fix this. If you don't have access to the Play Store, you can temporarily use Aurora Store and download Brave from there or use Fennec from the main F-Droid repos.

**Comment by AbhimanyuAryan** [comment]
any updates?

**Comment by murchu27** [comment]
+1 for this!

**Comment by kmk3** [comment]
Almost all replies after [this one][1] (and many more before it) do not contain
any new information related to the issue; they are either "+1"s, requests for
status or discussions.

Please be mindful that every new comment generates a notification to everyone
subscribed.  There are already 19 people who commented on this issue and
there's probably many more who are subscribed.

If you want to vote on this issue, just click on the emoji icon on [this
post][2] and click on :+1:.

If there had been any progress worthy of sharing, it would likely have already
been posted in here, so there is no point in asking for status updates.

If you want to discuss this, there are many threads already on
<https://community.brave.com>, so just go there and pick one.

[1]: https://github.com/brave/brave-browser/issues/7027#issuecomment-829647709
[2]: https://github.com/brave/brave-browser/issues/7027#issue-525811236

**Comment by Goldman90** [comment]
still no news about this? last comment was more than 2 years ago

**Comment by bsclifton** [comment]
Hi folks - a lot of people asking for updates; I've shared in a previous post, this is categorized as P5, meaning there is no planned work

This P5 status is accurate - we at Brave don't have any intention of doing this work at the moment. That's why there haven't been any updates. That could change in the future - we can keep this issue open as a catch-all for the request.  If we get enough traction, we can always reconsider and reprioritize

Thanks!

**Comment by jangorecki** [comment]
Thanks for the update.
The issue being the 7th most upvoted open issue (from all ~5000) is still giving good chances for having it implemented at some point I believe.

**Comment by Mhowser** [comment]
I'm still very interested in this, it should be reprioritized since Google might decide to remove Brave from the play store on account of its Ad-blocking features.

**Comment by quazar-omega** [comment]
Are they actually planning to?

**Comment by marek22k** [comment]
I am also very interested in a release in F-Droid!

**Comment by Mhowser** [comment]
> Are they actually planning to?

@quazar-omega not currently, but google has made advances toward greater control over the internet by indroducing Web Environment Integerity. So I wouldn't be surprised if they stop tolerating adblocking apps in their store as well.

**Comment by rusty-snake** [comment]
FTR: We have 32bit apks (the MonoARM64 runs 32 bit too) attached to github releases. They can be side loaded or installed by an app like Obtainium.

So you can install brave w/o GPlay

**Comment by jech** [comment]
I agree that having Brave in the official F-Droid repository might be onerous, due to the requirement to rebuild the application by F-Droid.  However, there are other solutions:

  - Bromite offers the ability to add the Bromite repository (curated by the Bromite authors) to the F-Droid client: https://www.bromite.org/fdroid
  - FairEmail periodically checks the GitHub releases page, and offers to upgrade if a newer version is found: https://github.com/M66B/FairEmail/blob/master/FAQ.md#faq147

Either of these solutions would be fine for most LineageOS users, and could probably be imtivaplemented in an hour by a sufficiently motivated Brave developer.

**Comment by marek22k** [comment]
> I agree that having Brave in the official F-Droid repository might be onerous, due to the requirement to rebuild the application by F-Droid.

These criteria are exactly why I would like to have Brave in F-Droid. That way I could be sure that it doesn't include trackers and no proprietary components.

**Comment by jech** [comment]
> These criteria are exactly why I would like to have Brave in F-Droid. That way I could be sure that it doesn't include trackers and no proprietary components.

I hold no strong opinions either way, but there might exist good reasons why the Brave developers might not want to give up control to F-Droid.  You might want to read this, but please take it with a pinch of salt: https://privsec.dev/posts/android/f-droid-security-issues/

**Comment by marek22k** [comment]
Probably it would exceed the scope of this feature request, however if the issue is that F-Droid distributes a different version than Brave (different APK signature), [Reproducible Builds](https://f-droid.org/en/docs/Reproducible_Builds/) would be a solution.

But yes, F-Droid is not perfect.

**Comment by FusionDragon** [comment]
Ever since google started doing that youtube ad stuff i've been trying my best to push it out of my life as much as possible, Brave would benefit from having it's app on fdroid without a doubt.

**Comment by inson1** [comment]
+1

**Comment by et-ness** [comment]
> We install Brave to get away from Google, and we have to download it from Play Store 😆

if you want go away from google, you must install firefox. Chromium is developed from Google.
On fdroid there is fennec, it's a firefox with tracking code removed.

**Comment by rusty-snake** [comment]
> if you want go away from google, you must install firefox. Chromium is developed from Google.

Guess how develops the platform (Android/AOSP).

**Comment by Mhowser** [comment]
You can download the apk from github releases without needing the play store. We should keep this on topic and figure out a solution for this issue.

**Comment by rusty-snake** [comment]
https://github.com/brave/brave-browser/issues/7491#issuecomment-541464445

**Comment by Mhowser** [comment]
I have moved on to the Fennec browser, which is in fdroid already and has great extension support. I would be great if the team could put some effort into this for other users.

**Comment by Mhowser** [comment]
Yeah I do agree there are some issues with Fennec, since I don't have any google services on my device I need my apps to be on the fdroid ecosystem. I also wanted to personally try Fennec out for myself anyway since many people have recommended it.

**Comment by jech** [comment]
Obtainium has improved a lot over the last year (they've recently released 1.0, by the way).  I've been sing it to keep my copy of Brave up to date for the last year or so with no issued.

I know that my experience does not necessarily apply to everyone, but thanks to Obtainium, I no longer need the Brave developers to do anything they aren't already doing.

https://github.com/ImranR98/Obtainium

**Comment by RasheedAZ** [comment]
> Obtainium has improved a lot over the last year (they've recently released 1.0, by the way). I've been sing it to keep my copy of Brave up to date for the last year or so with no issued.
> 
> I know that my experience does not necessarily apply to everyone, but thanks to Obtainium, I no longer need the Brave developers to do anything they aren't already doing.
> 
> https://github.com/ImranR98/Obtainium

How do you filter out Nightly and Beta releases? I get Nightly when I add this to Obtainium because of https://github.com/brave/brave-browser/issues/30270 and other people too: https://github.com/ImranR98/Obtainium/issues/20#issuecomment-1962499126

**Comment by jech** [comment]
I'm one version late, as described in the bug report.  Interesting.

**Comment by Aadam-Marshall** [comment]
+1. Brave's dependency on GMS is frustrating and breaks the UX of people who don't have GMS installed on their phones. Maybe Brave could release a separate build with all the Google's stuff ripped out (maintaining a separate build is a huge PITA, I know...), or just make the dependency on Google Play Services optional: during the installation, Brave would "see" that a user has no GMS installed on their phone, and would automatically disable all features that require GMS. Brave is a beautiful browser, but can't really be very private until the dependency on GMS will be removed from it. I'm patiently waiting for this to happen.

**Comment by jech** [comment]
> +1. Brave's dependency on GMS is frustrating and breaks the UX of people who don't have GMS installed on their phones.

I think there's a confusion. Brave does not depend on Google Play Services, it works just fine on a fresh LineageOS install.  The issue is with Brave not being in the default FDroid repository, which means that it needs to be installed by other means, either manually or by using something like Obtainium.

**Comment by fmarier** [comment]
We are currently working on a third-party F-Droid repository that users will be able to add to the F-Droid app in order to install Brave and keep it up to date without requiring the Google Play app.

**Comment by knuxyl** [comment]
> We are currently working on a third-party F-Droid repository that users will be able to add to the F-Droid app in order to install Brave and keep it up to date without requiring the Google Play app.

I don't believe this is a valid solution to the problem. There already exists alternative auto update/install software for apps from github so this only accomplishes bypassing fdroids software compliancy requirements and still being available through their app.
A proper solution would be the remove GMS dependency and any other proprietary code and provide reproducible builds to the community through fdroids official server.
I'm not quite sure what the exact issue with removing these dependencies is but I'm sure there's also some privacy implications with keeping it, especially considering the "backdoor" in the meetings extension.
I was hoping to move my entire stack to this browser but not being on fdroid is a big no go for me.
For reference I have debian on 4 workstations and lineageos for microg on 3 phones. Currently using librewolf and mull.

**Comment by fmarier** [comment]
> There already exists alternative auto update/install software for apps from github

You're talking about [FFUpdater](https://f-droid.org/en/packages/de.marmaro.krt.ffupdater/) I presume?

**Comment by knuxyl** [comment]
I was saying just in general because there are multiple. The one that came to mind was actually [Obtainium](https://f-droid.org/packages/dev.imranr.obtainium.fdroid/)

**Comment by Aadam-Marshall** [comment]
@knuxyl

Including the app into F-Droid requires removing proprietary libraries, which is Google Play Services. If Brave ever were to be included into F-Droid, the developers are going to *have* to remove GPS dependency. Hence, including Brave into F-Droid solves the problem of removing GPS dependency. If the devs insist on publishing a FOSS flavor of Brave into F-Droid and not just releasing a separate binary on GitHub releases for whatever reason - fine, we are getting a non-GPS Brave version either way.

**Comment by knuxyl** [comment]
> [@knuxyl](https://github.com/knuxyl)
> 
> Including the app into F-Droid requires removing proprietary libraries, which is Google Play Services. If Brave ever were to be included into F-Droid, the developers are going to _have_ to remove GPS dependency. Hence, including Brave into F-Droid solves the problem of removing GPS dependency. If the devs insist on publishing a FOSS flavor of Brave into F-Droid and not just releasing a separate binary on GitHub releases for whatever reason - fine, we are getting a non-GPS Brave version either way.

Yeah that's good and all but removes all trust, at least for me, that the work was actually done to remove all the dependencies. I'm a hobby dev and I'm not going to look through all the source code to verify it's not spyware and doesnt have backdoors, assuming I'd even recognize it. For me that route completely ignores the reasoning for me wanting an fdroid release. It's another layer of insurance that the software is "safer" to use and doesn't include parts that I don't want running on my hardware. That's the purpose of Fdroid, it's not just a file host.

**Comment by TheGerman0** [comment]
> [@knuxyl](https://github.com/knuxyl)
> 
> Including the app into F-Droid requires removing proprietary libraries, which is Google Play Services. If Brave ever were to be included into F-Droid, the developers are going to _have_ to remove GPS dependency. Hence, including Brave into F-Droid solves the problem of removing GPS dependency. If the devs insist on publishing a FOSS flavor of Brave into F-Droid and not just releasing a separate binary on GitHub releases for whatever reason - fine, we are getting a non-GPS Brave version either way.

@Aadam-Marshall No, that would be only requiered when they would add it to the official F-Droid Repo. In your own Repo you can do what you want, your App doesn't even has to be Open-Source.

**Comment by jech** [comment]
> That removes all trust, at least for me, that the work was actually done to remove all the dependencies.

As far as I know, there are no Chromium derivatives in F-Droid's default repository.  Removing all proprietary Google code from Chromium is a huge task, and it's most probable that the F-Droid volunteers don't have the manpower to do the verification.  I think it's very unlikely that a version of Brave will be accepted into F-Droid.

If you require a web browser that is accepted into F-Droid, you're going to need to switch to a Firefox derivative.

**Comment by numselli** [comment]
> We are currently working on a third-party F-Droid repository that users will be able to add to the F-Droid app in order to install Brave and keep it up to date without requiring the Google Play app.

What is the status of this? On the Android General project board this issue is marked as In Progress. It seems like at some point that project board was closed.

**Comment by oajara** [comment]
We’re putting on the finishing touches—it should be ready to launch soon.

**Comment by fmarier** [comment]
You can now find our official F-Droid repositories here: https://github.com/brave/brave-browser/wiki/Installing-Brave-on-Android

This means that while Brave is not available in the F-Droid **default repository**, it is possible to add our official repositories to the F-Droid **app** and take advantage of automatic updates without the need for Google Play.

**Comment by fmarier** [comment]
We announced it on our blog: https://brave.com/blog/f-droid/

**Comment by knuxyl** [comment]
This is a great step forward but I would still prefer it to be in the official fdroid repositories because they will unbiasely notify users of all privacy or source code issues. Until this is done I have to stick with Fennec and Mull.

**Comment by rusty-snake** [comment]
> will unbiasely notify users of all privacy or source code issues

They are biased.
They don't detect all, even obvious, issues.

Having an own repo for brave is a feature.

**Comment by knuxyl** [comment]
> > will unbiasely notify users of all privacy or source code issues
> 
> They are biased. They don't detect all, even obvious, issues.
> 
> Having an own repo for brave is a feature.

I don't see that as true at all, they don't actually develop any of the apps hosted. What evidence or behavior from them suggests what you are saying is true? This is a genuine question, I'd like to know. Goal is privacy, not blindly supporting a platform.

**Comment by Aadam-Marshall** [comment]
> in the official fdroid repositories

Again, for Brave to be in the F-Droid repositories, the developers will have to de-Google Brave (it's unknown if/when it will be done). That's not what they did. They just provided another comfy way of downloading Brave binaries besides GitHub. And surely it would be perfect if at least the Sync feature worked without Google Play Services. I don't think many "de-Googled" users need something besides Sync.

**Comment by knuxyl** [comment]
> will have to de-Google Brave

well that's what I'm waiting for. I don't use sync with any google services except phone contacts but I can see that being a problem with password management. There was an open source alternative for doing this with google servers but it was a mess and complicated. I use a microg lineageos without any google spyware running and I'd just prefer a no spying browser. If this is tied in with google as much as everyone is making it seem then I really don't see the point in this browser? I guess I'm just not that interested to learn which niche this project fits into, I assumed it was a privacy focused browser.

**Comment by wenj7488** [comment]
如果可以，你可以幫我嗎？我是一個完全不懂的人，但他用串聯跟飛機還有機器人來做扶助，，相連之帳號還配合到外國，，藏很深，，很難找，我也一直找找出原頭，但這全是用正版的器材，，有谷歌，微軟，三星

knuxyl ***@***.***> 於 2025年6月28日 週六 21:11 寫道：

**Comment by fmarier** [comment]
> And surely it would be perfect if at least the Sync feature worked without Google Play Services. I don't think many "de-Googled" users need something besides Sync.

We are aware that the QR code scanner doesn't work without Google Play Services (https://github.com/brave/brave-browser/issues/11477), however Sync itself works fine. You just need to pair the two devices using the code words.

**Comment by fmarier** [comment]
And just to be extra-clear, our Sync does not use Google services. It relies instead on a custom Open Source backend we wrote: https://github.com/brave/go-sync

**Comment by ghost** [comment]
@fmarier Can you also add Changelogs to updates in F-Droid as soon as you added them on GitHub?

**Comment by fmarier** [comment]
@UsrnmeX Could you please file a new issue for this?

We'll have to look into how to add that into our automated pipeline.

**Comment by ghost** [comment]
@fmarier I created an issue (https://github.com/brave/brave-browser/issues/48089)

Can you please also upload the latest Brave update to F-Droid? I don't think an extra issue is needed, just noticed it is still not there. Probably someone forgot it because the Desktop and Android updates have different version numbers this time.

**Comment by fmarier** [comment]
> Can you please also upload the latest Brave update to F-Droid?

I see the same version on the Google Play store and F-Droid and so it's up to date. Desktop and Android don't necessarily release at the same time.

**Comment by ghost** [comment]
> I see the same version on the Google Play store and F-Droid and so it's up to date. Desktop and Android don't necessarily release at the same time.

Probably because of the staged rollout, but Play Store shows an update from 31. July. In the last versions, these made no difference for F-Droid, there was no staged rollout, the version was instantly there. When you changed that it is okay, it is probably even a better solution to upload it not before the staged rollout in the Play Store finished (F-Droid itself doesn't support something like staged rollouts AFAIK) when you really upload it to F-Droid as soon it is done. That would be tomorrow as per https://github.com/brave/brave-browser/wiki/Brave-Release-Schedule#android-rollout-process
For me personally it is no problem. I know there is definitely a new update, (also mentioned here on GitHub in the Android changelog) but some F-Droid users won't know otherwise.

**Comment by ghost** [comment]
1.80.126 is now definitely available in Play Store for everyone, and still not in F-Droid.
